jenkins_files/ex03/Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+pipeline {
+    agent any
+
+    stages {
+        stage('Build') {
+            steps {
+                echo env.STAGE_NAME
+            }
+        }
+        stage('test') {
+            steps {
+                echo env.STAGE_NAME
+            }
+        }
+    }
+}
```

Comment:
add the cron job expression here too

Reply:
i have added screen shot of cron job . is that okay?